README.md

```diff
@@ -1,3 +1,15 @@
+## Pre-requisites
+
+**Hook**
```

Comment:
We should only use bold for UI text. I recommend using a H3 here or just normal text.